SETUP.md

```diff
@@ -0,0 +1,8 @@
+## Dev environment installation
+
+1. Install [docker](https://docs.docker.com/get-docker/)
+2. Run the following command to get a postgresql database running - `docker run --name <container-name> -p 5432:5432 -e POSTGRES_PASSWORD=<password> -d postgres:14`. Note that default user and database is postgres.
```

Comment:
The psql command only installs the psql client. I've used Docker for the database because it is less of a hassle but yes postgres database can be installed in whatever way is convenient.

Reply:
Got it, makes sense.